packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -216,41 +201,24 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ReinitializableBase
     /// @notice Thrown when the caller is not authorized to call the function.
     error OptimismPortal_Unauthorized();
 
-    /// @notice Thrown when the wrong proof method is used.
-    error OptimismPortal_WrongProofMethod();
-
-    /// @notice Thrown when a super root proof is invalid.
-    error OptimismPortal_InvalidSuperRootProof();
-
-    /// @notice Thrown when an output root index is invalid.
-    error OptimismPortal_InvalidOutputRootIndex();
-
-    /// @notice Thrown when an output root chain id is invalid.
-    error OptimismPortal_InvalidOutputRootChainId();
-
-    /// @notice Thrown when trying to migrate to the same AnchorStateRegistry.
-    error OptimismPortal_MigratingToSameRegistry();
-
     /// @notice Semantic version.
-    /// @custom:semver 4.6.0
+    /// @custom:semver 4.7.0
     function version() public pure virtual returns (string memory) {
-        return "4.6.0";
+        return "4.7.0";
     }
 
     /// @param _proofMaturityDelaySeconds The proof maturity delay in seconds.
-    constructor(uint256 _proofMaturityDelaySeconds) ReinitializableBase(2) {
+    constructor(uint256 _proofMaturityDelaySeconds) ReinitializableBase(3) {
```

Comment:
Comment only: Given my understanding that it should be possible to upgrade to this contract from either U15 or U16, `3` is the correct value here.